test/core/priors.jl

```diff
@@ -122,15 +123,15 @@ end
 
     @testset "Test vanilla (ie. non-preconditioned)" begin
         Ah, Qh = PNDE.discretize(prior, h)
-        Qh = PNDE.apply_diffusion(Qh, σ^2)
+        Qh = PNDE.apply_diffusion(Qh, σ^2*Eye(d))
```

Comment:
Instead of these, define a `\Gamma` somewhere and use that.

Reply:
And then ideally do it for the FillArray Gamma, but also for a standard Diagonal one